windowPreview.js

```diff
@@ -370,7 +371,10 @@ class WindowPreviewMenuItem extends PopupMenu.PopupBaseMenuItem {
 
     _getWindowPreviewSize() {
         let mutterWindow = this._window.get_compositor_private();
-        let [width, height] = mutterWindow.get_size();
+        const [width, height] = mutterWindow?.get_size() ?? [0, 0];
```

Comment:
```
if (!mutterWindow?.get_texture()) return [0, 0, 0];

const [width, height] = mutterWindow.get_size();

if (!width || !height) ...
```
Can windows have a 0x0 size once they are mapped? :thinking: 

I don't think get_size itself can return `null`, so it may be clearer to separate the check if it is necessary.

Reply:
Done